dispatcher.go

```diff
@@ -206,6 +206,16 @@ func (d *Dispatcher) Inbounds() Inbounds {
 	return inbounds
 }
 
+// Outbounds returns a copy of the map of outbounds for this RPC object.
+// The outbounds are already wrapped with middleware
+func (d *Dispatcher) Outbounds() Outbounds {
```

Comment:
We can assert on the existence of Outbounds through ClientConfig calls right?  Since this is only for tests, do we need this function?